src/Controller/TreeController.php

```diff
@@ -102,6 +102,10 @@ public function reorderAction(Request $request)
             return new JsonResponse(array('Parameters parent, dropped and target has to be set to reorder.'), Response::HTTP_BAD_REQUEST);
         }
 
+        if (in_array($position, array('over', 'child'))) {
+            return new JsonResponse(array('Can not resort into a collection.'), Response::HTTP_BAD_REQUEST);
```

Comment:
Is "resort" really what you mean here? http://www.wordreference.com/enfr/resort
the name of the action is "reorder" BTW